components/profile/sidebar.tsx

```diff
@@ -7,80 +17,93 @@ import {
 	CardTitle,
 } from "@/components/ui/card";
 import { cn } from "@/lib/utils";
+import { ArrowUpRight, CheckCircle2 } from "lucide-react";
 import Link from "next/link";
 
 const SideBar = () => {
+	const [anonAadhaar] = useAnonAadhaar();
+
+	useEffect(() => {
+		console.log("Anon Aadhaar status: ", anonAadhaar.status);
+	}, [anonAadhaar.status]);
+
 	return (
 		<section className="flex flex-col gap-4 md:col-span-1 md:row-span-2">
-			<Card
-				className={cn("rounded-3xl bg-vd-beige-100 shadow-none border-none")}
-			>
+			<Card className="bg-vd-beige-100 rounded-3xl shadow-none border-none">
+				<AnonAadhaarProvider _useTestAadhaar={true}>
+					<CardHeader>
+						<img
+							src="/teapot.svg"
+							alt="teapot that looks like a building"
+							className="mx-auto px-6 w-40 h-auto my-4"
+						/>
+						<CardTitle className="text-center text-lg md:text-xl">
+							{anonAadhaar.status === "logged-in"
+								? "You're a local advocate!"
+								: "Log in to become a local advocate"}
+						</CardTitle>
+						<CardDescription className="text-center text-base">
+							{anonAadhaar.status === "logged-in"
+								? "Verified citizens can evaluate impacts and vote on VoiceDeck, shaping their community's future."
+								: "Verify with Aadhaar to access community impact evaluation and voting on VoiceDeck."}
+						</CardDescription>
```

Comment:
@thebeyondr were you able to see this change in the ui previously?
the changed `anonAadhaar.status` isn't triggering the conditional ui update.
I'm logged in here:
<img width="727" alt="Screenshot 2024-04-08 at 12 08 51 PM" src="https://github.com/VoiceDeck/app/assets/79813968/231964f7-bc72-4c83-8fe1-2293aa00c6bd">

Reply:
yes, this was fine... I'll look into it

Reply:
Fixed in https://github.com/VoiceDeck/app/pull/89/commits/27caa9dceac03a1439091ef50f5cf66e1a52fa2f. The client-side refactor caused the `useAnonAadhaar` hook to be used outside of the provider it depends on: so it could not receive updates

Reply:
👍 thank you!